pom.xml

```diff
@@ -84,7 +84,7 @@
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>target-platform-configuration</artifactId>
         <configuration>
-          <executionEnvironment>JavaSE-11</executionEnvironment>
+          <executionEnvironment>JavaSE-17</executionEnvironment>
```

Comment:
Maybe the `release` parameter of the `tycho-compiler-plugin` should be adjusted as well?

Reply:
If you know what to do and where, feel free to provide a PR :-)

Reply:
> If you know what to do and where, feel free to provide a PR :-)

Done in https://github.com/eclipse-platform/eclipse.platform.swt.binaries/pull/37 
Just mentioned it before as a reminder for myself or to give you or somebody else the chance to add if earlier if desired. :)